CRM/Event/WorkflowMessage/ParticipantTrait.php

```diff
@@ -67,6 +67,10 @@ trait CRM_Event_WorkflowMessage_ParticipantTrait {
    */
   protected $participantContacts;
 
+  private function isContributionComponentEnabled(): bool {
+    return CRM_Core_Component::isEnabled('CiviContribute');
```

Comment:
So we did switch to checking for entity present in tokens - ie https://github.com/civicrm/civicrm-core/pull/27201/files#diff-1577dbbf2e3373be5f6fb29637b0ec2bee12390aabb24997d951b3509e44b7f3R61

perhaps that is more important in tokens as loaded earlier

Reply:
Just to be clear the action object provider is a known list of APIv4 entities it isn't specific about if an entity is in tokens or not but I suppose I could switch this to that style if we wanted it like that

Reply:
IMO it's best to get away from using `CRM_Core_Component` as much as possible; as future-proofing and to make it easier to eventually get rid of it.

Reply:
ok I have updated now and also renamed function to remove the word Component as well